Documentation/Books/AQL/Operations/Insert.md

```diff
@@ -100,6 +100,22 @@ FOR i IN 1..1000
   } INTO users OPTIONS { overwrite: true }
 ```
 
+
+In contrast to the MMFiles engine, the RocksDB engine does not require collection-level
+locks. Different write operations on the same collection do not block each other, as
+long as there are no _write-write conficts_ on the same documents. From an application
+development perspective it can be desired to have exclusive access on collections,
+to simplify the development. 
```

Comment:
At the cost of blocking other operations until done? (Add something like "exclusive access means writes block reads" yada yada)

Reply:
no reads are never blocked